appinfo/info.xml

```diff
@@ -21,7 +21,7 @@
 	<category>security</category>
 	<bugs>https://github.com/ChristophWurst/twofactor_admin/issues</bugs>
 	<dependencies>
-		<php min-version="7.4" max-version="8.1"/>
+		<php min-version="8.0" max-version="8.2"/>
```

Comment:
7.4 must not be dropped before Nextcloud 25 support is gone too. Else the app store will send updates to Nextcloud 25 and PHP7.4 installations and they will :boom:

Reply:
Oh, I believed it will stop updates for instances not running PHP 8.0.
I'll fix this!